sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md

```diff
@@ -12,7 +12,7 @@
 ## 1.3.0 (2023-08-15)
 
 ### Features Added
-- Updated to support ANF api-version 2022-09-01
```

Comment:
Just let you know, this change will not take effect since the changelog has been out with the already released package version.